_data-prepper/pipelines/configuration/processors/delete-entries.md

```diff
@@ -33,10 +43,13 @@ To get started, create the following `pipeline.yaml` file:
 pipeline:
   source:
     ...
-  ....  
+  ....
   processor:
     - delete_entries:
-        with_keys: ["message"]
+        entries:
```

Comment:
I think we should have a completely different example for this rather than sharing the single example.

Reply:
Thanks! Moving on to editorial review then.